src/analyzer/protocol/ssl/events.bif

```diff
@@ -589,10 +627,11 @@ event ssl_plaintext_data%(c: connection, is_client: bool, record_version: count,
 ## content_type: message type as reported by TLS session layer. Not populated for
 ##               SSLv2.
 ##
-## length: length of the entire message.
+## length: encrypted payload length of the message.
```

Comment:
This doesn't sound right to me. Maybe this?

```suggestion
## length: length of the encrypted payload in the message.
```

Reply:
That makes more sense. I probably will make it "length of the encrypted payload in the record" - which is even more correct, if that is ok with you :)